examples/hook-and-listen.py

```diff
@@ -4,13 +4,22 @@
 
 
 def on_event(args):
+    """Callback for keyboard and mouse events"""
     if isinstance(args, KeyboardEvent):
         if args.current_key == 'A' and args.event_type == 'key down' and 'Lcontrol' in args.pressed_key:
             print("Ctrl + A was pressed");
 
         if args.current_key == 'K' and args.event_type == 'key down':
             print("K was pressed");
 
+        if args.current_key == 'M' and args.event_type == 'key down' and 'U' in args.pressed_key:
+            hk.unhook_mouse()
+            print("Unhook mouse")
```

Comment:
`from __future__ import print_function` is required at the beginning of the module (before all other imports) to make it working on both Py2 and Py3.